etl/churn.kp/knowledge.md

```diff
@@ -496,7 +486,7 @@ def compute_churn_week(df, week_start, bucket, prefix):
                 x.get('start_version', 'unknown'),
                 x.get('sync_usage', 'unknown'),
                 x.get('current_version', 'unknown'),
-                x.get('current_week', 'unknown'),
+                x.get('current_week', -1),
```

Comment:
The defaults were intended to display reasonable values in the CSV output in previous iterations of the code. The parquet dataset might be better off using null, since it actually means "unknown" and can be used for any data type. We could change all of these to be null if data is missing, then use `df.fillna('unknown')` in the CSV generation code. What do you think?